pepper-apis/src/main/java/org/broadinstitute/ddp/export/collectors/FileQuestionFormatStrategy.java

```diff
@@ -0,0 +1,48 @@
+package org.broadinstitute.ddp.export.collectors;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.broadinstitute.ddp.content.HtmlConverter;
+import org.broadinstitute.ddp.elastic.MappingUtil;
+import org.broadinstitute.ddp.model.activity.definition.question.FileQuestionDef;
+import org.broadinstitute.ddp.model.activity.instance.answer.FileAnswer;
+
+/**
+ * Rule:
+ * - single column for question response
+ * - value formatted as text (the file upload guid)
+ * - null value (or missing file upload) results in empty cell
```

Comment:
I'm not sure what's the best way to represent file data in the CSV export file. I just went with something simple like a single column with the file upload guid. Hopefully study staff can map that guid back to the actual file in DSM data viewer.